internal/vrf/vrf.go

```diff
@@ -0,0 +1,53 @@
+package vrf
+
+import (
+	"github.com/equinix/metal-cli/internal/outputs"
+	"github.com/packethost/packngo"
+	"github.com/spf13/cobra"
+)
+
+type Client struct {
+	Servicer Servicer
+	Service  packngo.VRFService
+	Out      outputs.Outputer
+}
+
+func (c *Client) NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     `vrf`,
+		Aliases: []string{"vrf"},
+		Short:   "VRF operations : create, TODO: make other commands.",
+		Long:    "Experimental VRF function"
+
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if root := cmd.Root(); root != nil {
+				if root.PersistentPreRun != nil {
+					root.PersistentPreRun(cmd, args)
+				}
+			}
+			c.Service = c.Servicer.API(cmd).ProjectVRFs //Not sure about this endpoint
```

Comment:
@irusoeqx if you're up for it, you can convert your PR to use `metal-go` by rebasing on the latest commit from the `metal-cli` `main` branch.  As of that commit, there is a [`MetalAPI`](https://github.com/equinix/metal-cli/blob/main/internal/cli/root.go#LL192C18-L192C26) function that provides a `metal-go` client.

With `metal-go`, this line would become something like

```go
c.Service = c.Servicer.MetalAPI(cmd).VRFsAPI
```

Docs for `VRFsApi` are here (note that the code examples in docs are auto-generated and serve as examples of syntax, not of fully-working code): https://github.com/equinix-labs/metal-go/blob/main/docs/VRFsApi.md

Reply:
Yes il do that, I just need a little time.